cocos/ui/UIAbstractCheckButton.h

```diff
@@ -0,0 +1,220 @@
+/****************************************************************************
+Copyright (c) 2013-2014 Chukong Technologies Inc.
+
+http://www.cocos2d-x.org
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+****************************************************************************/
+
+#ifndef __UIABSTRACTCHECKBUTTON_H__
+#define __UIABSTRACTCHECKBUTTON_H__
+
+#include "ui/UIWidget.h"
+#include "ui/GUIExport.h"
+
+/**
+ * @addtogroup ui
+ * @{
+ */
+NS_CC_BEGIN
+class Sprite;
+
+namespace ui {
+    
+/**
+ *  AbstractCheckButton is a specific type of two-states button that can be either checked or unchecked.
+ */
+class CC_GUI_DLL AbstractCheckButton : public Widget
+{
+    
+public:
+    /**
+     * Default constructor.
+     *
+     * @lua new
```

Comment:
@neokim  please remove `@lua new` here. The AbstractCheckBox is a abstract class and it can't be initialized in C++, not mentioned in Lua side. So the docs here is wrong.